SourceX/DiabloUI/selgame.h

```diff
@@ -1,8 +1,9 @@
 #pragma once
+#include "all.h"

```

Comment:
```suggestion
#include "gendung.h"
```

Reply:
Felt it did not belong, all is already commonly included, so pushed resolving this for when we clean up all.h etc

Reply:
Thought that might be the case.